main.py

```
@@ -0,0 +1,179 @@
+
+Pregunta: {user_query}
+"""
+
+    # 3. Genera la respuesta usando el modelo Gemini
+    try:
+        responses = model.generate_content([Part.from_text(prompt)])
+        if responses.candidates:
+            gemini_response = responses.candidates[0].content.parts[0].text
+        else:
+            gemini_response = "No se obtuvo respuesta del modelo Gemini."
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)